eng/Versions.props

```diff
@@ -1,12 +1,26 @@
-﻿<Project>
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MajorVersion>3</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseLabel>preview</PreReleaseLabel>
+    <PreReleaseVersionLabel>$(PreReleaseLabel)</PreReleaseVersionLabel>
     <OfficialBuildId Condition="'$(OfficialBuildId)' == ''">$(BUILD_BUILDNUMBER)</OfficialBuildId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RestoreSources>
```

Comment:
We should remove this if we are going to have a non-empty NuGet.config.

Reply:
I ended up needing both, but that might be because of how testing for MSBuild is done in Razor.